apps/docker-compose.yml

```diff
@@ -5,6 +5,7 @@ version: '3'
 services:
   phoenix_web:
     restart: unless-stopped
+    dns: 1.1.1.1
```

Comment:
I don't think this is wise

Reply:
why not? We do need to give it some DNS server otherwise we can't upload files to spaces